nixos/tests/all-tests.nix

```
@@ -649,6 +650,7 @@ in {
   systemd-initrd-btrfs-raid = handleTest ./systemd-initrd-btrfs-raid.nix {};
   systemd-initrd-luks-fido2 = handleTest ./systemd-initrd-luks-fido2.nix {};
   systemd-initrd-luks-keyfile = handleTest ./systemd-initrd-luks-keyfile.nix {};
+  systemd-initrd-luks-empty-passphrase = handleTest ./systemd-initrd-luks-empty-passphrase.nix {};
```

This file doesn't exist. I'm guessing it's supposed to be like this?

```suggestion
  systemd-initrd-luks-empty-passphrase = handleTest ./initrd-luks-empty-passphrase.nix { systemdStage1 = true; };
```

---

Yeah my mistake, will fix